.registryrc

```diff
@@ -1 +1 @@
-26e513e8fe567428cb58e7634e0ed8c659430707
+3b7433b52d2f520e6b104f98d3da04f26b310a1f
```

Comment:
shouldn't we merge https://github.com/hyperlane-xyz/hyperlane-registry/pull/593 before merging this pr